client/plots/boxplot/ViewModel.ts

```diff
@@ -9,33 +20,58 @@ export class ViewModel {
 	readonly horizPad = 120
 	/** For outliers, set a radius rather than using the default. */
 	readonly outRadius = 5
+	/** Increasing padding to space out the boxplots and determine position */
+	incrTopPad = 40
 	constructor(config: any, data: any, settings: BoxplotSettings) {
-		for (const plot of data.plots) {
-			if (!plot.color) plot.color = config?.term2?.term?.values?.[plot.seriesId]?.color || settings.color
-			if (plot.boxplot.out.length) {
-				const maxOut = plot.boxplot.out.reduce((a, b) => Math.max(a.value, b.value))
-				if (maxOut && maxOut.value > data.absMax) data.absMax = maxOut.value
-				plot.boxplot.radius = this.outRadius
-			}
-		}
+		this.settings = settings
+		this.config = config
+
+		if (!data || !data.plots.length) return
+		const viewData: any = structuredClone(data)
+
+		const totalLabelWidth = viewData.maxLabelLgth * 4 + this.settings.labelPad + this.horizPad
+		const totalRowHeight = this.settings.rowHeight + this.settings.rowSpace
 
 		/** Add more plot dimensions here
 		 * Eventually should calculate the difference between vertical and
 		 * horizontal orientation.
-		 *
-		 * maybe return as const? not in data?
 		 */
 
-		const totalLabelWidth = data.maxLabelLgth + settings.labelPad + this.horizPad
-		const totalRowHeight = settings.rowHeight + settings.rowSpace
+		viewData.plotDim = {
```

Comment:
`viewData` should be declared as a class (or a type) instead of any so that the type checks will be performed inside of:

```
export class View {
	constructor(data: any, settings: BoxplotSettings, dom: BoxplotDom) {
```

It can be improved inside of the next PR.